src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java

```diff
@@ -23,6 +23,8 @@
     private boolean isRecipientNameHidden;
     private boolean isGiverTeam;
     private boolean isRecipientTeam;
+    private boolean isRecipientStudent;
+    private boolean isRecipientInstructor;
```

Comment:
It is possible to use an enum instead of multiple booleans? The three values (team, student, instructor) are mutually exclusive.